view_arangosearch.go

```diff
@@ -84,6 +87,24 @@ type ArangoSearchViewProperties struct {
 	// are indexed in thie view.
 	// The key of the map are collection names.
 	Links ArangoSearchLinks `json:"links,omitempty"`
+
+	// PrimarySort describes how individual fields are sorted
+	PrimarySort []ArangoSearchPrimarySortEntry `json:"primarySort,omitempty"`
+}
+
+// ArangoSearchSortDirection describes the sorting direction
+type ArangoSearchSortDirection string
+
+const (
+	ArangoSearchSortDirectionAsc  ArangoSearchSortDirection = "ASC"
+	ArangoSearchSortDirectionDesc ArangoSearchSortDirection = "DESC"
+)
+
+// ArangoSearchPrimarySortEntry describes an entry for the primarySort list
+type ArangoSearchPrimarySortEntry struct {
+	Field     string                     `json:"field,omitempty"`
+	Ascending *bool                      `json:"asc,omitempty"`
```

Comment:
why a boolean flag and a string for the direction? How do we ensure that these are set consistently?